src/models/DataSetStore.js

```diff
@@ -9,6 +22,7 @@ export default class DataSetStore {
         this.config = {
             categoryProjectsId: "MRwzyV0kXv9",
             categoryComboId: "GmXXE8fiCK5",
+            categoryComboDefaultId: "eCUWK7qTyeW",
```

Comment:
Indeed, it was used at some point in my tests,  but not anymore, removed: 793da63a